src/home/components/MyMenuList/MyMenuList.tsx

```diff
@@ -0,0 +1,57 @@
+import { IcArrowRightLine } from '@yourssu/design-system-react';
+import { useNavigate } from 'react-router-dom';
+import { useTheme } from 'styled-components';
+
+import {
+  StyledContainer,
+  StyledInnerContainer,
+  StyledMenuContainer,
+  StyledMenuListTitle,
+  StyledMenuTitle,
+  StyledListItem,
+} from './MyMenuList.style';
+
+export const MyMenuList = () => {
+  const theme = useTheme();
+  const navigate = useNavigate();
+
+  return (
+    <StyledContainer>
+      <StyledInnerContainer>
+        <StyledMenuListTitle>마이페이지</StyledMenuListTitle>
+        <StyledMenuContainer>
+          <StyledMenuTitle>계정 관리</StyledMenuTitle>
+          <StyledListItem>비밀번호 변경</StyledListItem>
+          <StyledListItem>로그아웃</StyledListItem>
+        </StyledMenuContainer>
+        <StyledMenuContainer>
+          <StyledMenuTitle>내 서랍장</StyledMenuTitle>
+          <StyledListItem
+            rightIcon={<IcArrowRightLine color={theme.color.buttonNormal} />}
+            onClick={() => navigate('/drawer/myDrawers', { state: 'MYDRAWER' })}
+          >
+            등록한 콘텐츠
+          </StyledListItem>
+          <StyledListItem
+            rightIcon={<IcArrowRightLine color={theme.color.buttonNormal} />}
+            onClick={() => navigate('/drawer/myDrawers', { state: 'STAR' })}
+          >
+            즐겨찾기한 콘텐츠
+          </StyledListItem>
```

Comment:
> 혹시 이렇게 특정 탭부터 보여주도록 할 수 있을까요?

`MyDrawer` 제작하면서 생각 못 했던 부분인데 일단 절 한번 올리고 시작하겠습니다

서랍장 DM에서도 UX를 위해 특정 탭부터 보여주는 기능이 필요하다는 얘기가 나와서,
둘 다 커버할 수 있는 방법으로다가... 제가 고민을 해봤는데요?
정리한 내용 캡쳐해서 전달 두립니다

![image](https://github.com/yourssu/Soomsil-Web/assets/87255462/c5c9d65b-06a9-4e9b-8994-4fc87a060bf2)

내용이 길지만 한 줄 요약하자면 쿼리 스트링으로 하겠다는 말이고요?
해당 방식으로 갔을 때 `MyMenuList` 코드가 다음과 같이 수정되어야 할 거 같아 코드 제안 남기고 갈게요 >.<

```suggestion
          <StyledListItem
            rightIcon={<IcArrowRightLine color={theme.color.buttonNormal} />}
            onClick={() => navigate('/drawer/myDrawers?tab=MYDRAWER')}
          >
            등록한 콘텐츠
          </StyledListItem>
          <StyledListItem
            rightIcon={<IcArrowRightLine color={theme.color.buttonNormal} />}
            onClick={() => navigate('/drawer/myDrawers?tab=STAR')}
          >
            즐겨찾기한 콘텐츠
          </StyledListItem>
```